rsmtool/modeler.py

```diff
@@ -433,17 +428,18 @@ def train_rebalanced_lr(self, df_train, feature_columns):
 
     def train_lasso_fixed_lambda_then_lr(self, df_train, feature_columns):
         """
-        Train `LassoFixedLambdaThenLR` (formerly empWtLasso) -
+        Train a "LassoFixedLambdaThenLR" model.
```

Comment:
I assume this pattern, which I've commented on several times will occur more throughout this file. And I recognize it's a bit of a stylistic choice. I will neglect the rest of the instances and you can decide if it's worthwhile to change them.

```suggestion
        Train a ``LassoFixedLambdaThenLR`` model.
```